tests/load/gcp/deploy.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+ZONE="us-central1-a"
+PROJECT="lma-light-load-testing"
+CREATE="gcloud compute instances create --project=$PROJECT --zone=$ZONE --network-interface=network-tier=PREMIUM,subnet=default --maintenance-policy=MIGRATE --service-account=921396806154-compute@developer.gserviceaccount.com --scopes=https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append --no-shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --reservation-affinity=any"
+START="gcloud compute instances start --project=$PROJECT --zone=$ZONE"
+SSH="gcloud compute ssh --project $PROJECT --zone $ZONE"
+
+# LMA spec
+NCPU=2
+GBMEM=4
+INSTANCE="n1-ssd-${NCPU}cpu-${GBMEM}gb"  # e.g. n1-ssd-2cpu-8gb
+AVALANCHE_INSTANCE="avalanche-$INSTANCE"  # e.g. avalanche-n1-ssd-2cpu-8gb
+URL_LMA="$INSTANCE.$ZONE.c.$PROJECT.internal"  # e.g. n1-ssd-2cpu-8gb.c.lma-light-load-testing.internal
+URL_AVALANCHE="$AVALANCHE_INSTANCE.$ZONE.c.$PROJECT.internal"  # e.g. avalanche-n1-ssd-2cpu-8gb.c.lma-light-load-testing.internal
+
+start_avalanche () {
+  if [[ $(gcloud compute instances describe "${AVALANCHE_INSTANCE}") ]]; then
+    echo "avalanche instance ${AVALANCHE_INSTANCE} already exists"
+    gcloud compute instances stop "${AVALANCHE_INSTANCE}"
+  else
+    echo "Creating avalanche instance ${AVALANCHE_INSTANCE}"
+    $CREATE "${AVALANCHE_INSTANCE}" --machine-type=e2-standard-4 --create-disk=auto-delete=yes,boot=yes,device-name=disk-"${AVALANCHE_INSTANCE}",image=projects/lma-light-load-testing/global/images/avalanche,mode=rw,size=10,type=projects/lma-light-load-testing/zones/us-central1-a/diskTypes/pd-balanced
+  fi
+
+  echo "Starting avalanche instance..."
+  $START "${AVALANCHE_INSTANCE}"
+  sleep 60
+  echo "Starting avalanche workers..."
+  $SSH "${AVALANCHE_INSTANCE}" -- 'for ((i = 1 ; i <= 20 ; i++)) ; do port=$((9000 + $i)); `nohup avalanche --metric-count=1000 --label-count=10 --series-count=10 --value-interval=30 --series-interval=36000000 --metric-interval=36000000 --port=$port 1>/dev/null 2>&1 &` ; done'
+}
+
+start_lma () {
+  # $1 = number of avalanche targets to scrape
+  NUM_SCRAPE_TARGETS=$1
```

Comment:
```suggestion
  NUM_SCRAPE_TARGETS="$1"
```

Always quote variables, please